examples/federation-workshop/cluster-2/safe/Deployment.yaml

```diff
@@ -72,6 +72,8 @@ spec:
               value: "/key/key.txt"
             - name: VSECM_ROOT_KEY_INPUT_MODE_MANUAL
               value: "false"
+            - name: VSECM_MANUAL_ROOT_KEY_UPDATES_K8S_SECRET
+              value: "false"
```

Comment:
Why do we have these in example workload manifests?

`VSECM_MANUAL_ROOT_KEY_UPDATES_K8S_SECRET` is for **VSecM Safe** to use; it will be meaningless for any other workload.

Reply:
Right! I'm going to remove it. Thank you